static/app/views/replays/detail/console/viewIssueLink.tsx

```diff
@@ -8,24 +8,27 @@ import {t} from 'sentry/locale';
 import space from 'sentry/styles/space';
 import {BreadcrumbTypeDefault, Crumb} from 'sentry/types/breadcrumbs';
 import useOrganization from 'sentry/utils/useOrganization';
+import {breadcrumbHasIssue} from 'sentry/views/replays/detail/console/utils';
 
 type Props = {
   breadcrumb: Extract<Crumb, BreadcrumbTypeDefault>;
+  className?: string;
 };
 
-function ViewIssueLink({breadcrumb}: Props) {
+function ViewIssueLink({breadcrumb, className}: Props) {
   const organization = useOrganization();
 
-  const {project: projectSlug, groupId, groupShortId, eventId} = breadcrumb.data || {};
-  if (!groupId || !groupShortId || !eventId) {
+  if (!breadcrumbHasIssue(breadcrumb)) {
     return null;
   }
+  const {project: projectSlug, groupId, groupShortId, eventId} = breadcrumb.data || {};
 
   const to = {
     pathname: `/organizations/${organization.slug}/issues/${groupId}/events/${eventId}/?referrer=replay-console`,
   };
   return (
     <StyledHovercard
+      className={className}
```

Comment:
What styles `ViewIssueLink`? Will it be confusing that it actually ends up styling the hovercard rather than the link?

Reply:
oh yeah. i was trying to flatten `IssueLinkWrapper` and `ViewIssueLink` but the `styled(..)\` ${ViewIssueLink} { float: left; }\`;` but exploded the linter. Somehow its ok having the wrapper in there though.

This className is from some refactoring and extra now, coming out.